dev/docker/gravitino/gravitino-dependency.sh

```diff
@@ -21,26 +21,65 @@ set -ex
 gravitino_dir="$(dirname "${BASH_SOURCE-$0}")"
 gravitino_dir="$(cd "${gravitino_dir}">/dev/null; pwd)"
 gravitino_home="$(cd "${gravitino_dir}/../../..">/dev/null; pwd)"
+gravitino_package_dir="${gravitino_dir}/packages/gravitino"
+gravitino_staging_dir="${gravitino_package_dir}/staging"
+gravitino_iceberg_rest_dir="${gravitino_package_dir}/iceberg-rest-server/libs/"
 
 # Build the Gravitino project
 ${gravitino_home}/gradlew clean build -x test
 
 rm -rf ${gravitino_home}/distribution
 # Prepare compile Gravitino packages
-${gravitino_home}/gradlew compileDistribution -x test
+${gravitino_home}/gradlew compileDistribution -x test -x rat
 
 # Removed old packages, Avoid multiple re-executions using the wrong file
 rm -rf "${gravitino_dir}/packages"
 mkdir -p "${gravitino_dir}/packages"
 
-cp -r "${gravitino_home}/distribution/package" "${gravitino_dir}/packages/gravitino"
+cp -r "${gravitino_home}/distribution/package" "${gravitino_package_dir}"
+mkdir -p "${gravitino_staging_dir}"
+
+echo "Start to download the jar package"
+
+mysql_driver="mysql-connector-java-8.0.27.jar"
+wget "https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.27/$mysql_driver" -O "${gravitino_staging_dir}/${mysql_driver}"
+cp "${gravitino_staging_dir}/${mysql_driver}" "${gravitino_package_dir}/catalogs/jdbc-mysql/libs/"
+cp "${gravitino_staging_dir}/${mysql_driver}" "${gravitino_package_dir}/catalogs/lakehouse-iceberg/libs/"
+cp "${gravitino_staging_dir}/${mysql_driver}" "${gravitino_iceberg_rest_dir}"
+cp "${gravitino_staging_dir}/${mysql_driver}" "${gravitino_package_dir}/libs/"
+
+pg_driver="postgresql-42.7.0.jar"
+wget "https://jdbc.postgresql.org/download/$pg_driver" -O "${gravitino_staging_dir}/${pg_driver}"
```

Comment:
Please keep the style and use `${pg_driver}`

Reply:
done